inc/customizer/customizer.php

```diff
@@ -555,10 +555,13 @@ function create_css_builder(){
 				}
 			}
 
-			if(isset($setting['callback'])) {
+			if ( isset( $setting['callback'] ) && isset( $setting['default'] ) ) {
```

Comment:
In general it's better to have as little conditional nesting as possible, so this could be simplified to 
```suggestion
			$val = get_theme_mod($id);
			if ( isset( $setting['callback'] ) && isset( $setting['default'] ) && $val != $setting['default'] ) {
```